tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
`make install` will bring deps + tox. But yes, you'd then *have* to run tox at least once in order to be able to run the linter or coverage, as they'd only be declared in tox with the suggested change. Probably not so good, to change, then.